ipn/ipnlocal/local.go

```diff
@@ -4624,6 +4631,15 @@ func (b *LocalBackend) applyPrefsToHostinfoLocked(hi *tailcfg.Hostinfo, prefs ip
 	hi.ShieldsUp = prefs.ShieldsUp()
 	hi.AllowsUpdate = envknob.AllowsRemoteUpdate() || prefs.AutoUpdate().Apply.EqualBool(true)
 
+	// count routes without exit node routes
+	routeCount := 0
+	for _, route := range hi.RoutableIPs {
+		if route.Bits() != 0 {
+			routeCount++
+		}
+	}
+	metricAdvertisedRoutes.SetInt(struct{}{}, int64(routeCount))
```

Comment:
It's still a bit awkward to pass an empty struct here. What do you think about usermetric exposing its own type for metrics without labels? It probably does not need to use `MultiLabelMap` at all, and can just be an expvar?